stories/list-unique-values.js

```diff
@@ -0,0 +1,22 @@
+import { storiesOf } from '@storybook/vue';
+
+import { ListUniqueValues } from '../dist/storybook/components';
+const stories = storiesOf('ListUniqueValues', module);
+
+stories.add('default', () => ({
+  components: { ListUniqueValues },
+  data(){
+    return {
+        options: [
+            {value: 'France', count: 12},
+            {value: 'UK', count: 1},
+            {value: 'China', count: 14},
+            {value: 'Framboise', count: 10},
+            {value: 'Italy', count: 23},
+        ],
+        filter: { column: "col1", operator: "in", value: []},
+        loaded: false,
+    }
+  },
+  template: `<div style="font-family: 'Montserrat', sans-serif;"><ListUniqueValues :loaded="loaded" :filter="filter" @input="filter = $event" :options="options"/></div>`,
```

Comment:
It's weird to add the `font-family` there, isn't it?
Either we decide that this component should use the font of it's parent component (nothing to do, my preference go there) or we enforce Montserrat but it needs to be indicated in the component style.
If what we want is to change the default storybook font to Montserrat, let's do it in .storybook/preview-head.html